src/ApiService/ApiService/OneFuzzTypes/Responses.cs

```diff
@@ -15,6 +18,23 @@ public record PendingNodeCommand(
     NodeCommandEnvelope? Envelope
 ) : BaseResponse();
 
+// TODO: not sure how much of this is actually
+// needed in the search results, so at the moment
+// it is a copy of the whole Node type
```

Comment:
Could we turn `BaseResponse` into an interface, add it on the existing `Node` record in `Model.cs`? The last thing to figure out would be the BaseResponse JsonConverter

Reply:
I think there's probably going to be things in `Node` (now or in future) that we don't want to reflect into the response here?

Reply:
I’ve also been bitten a lot of times in the past by sharing internal models as external ones, so I think I'll leave it like this for now.